vnext/DeforkingPatches/ReactCommon/turbomodule/core/TurboModuleBinding.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "TurboModuleBinding.h"
+
+#include <stdexcept>
+#include <string>
+
+#include <ReactCommon/LongLivedObject.h>
+#include <cxxreact/SystraceSection.h>
+
+using namespace facebook;
+
+namespace facebook {
+namespace react {
+
+/**
+ * Public API to install the TurboModule system.
+ */
+TurboModuleBinding::TurboModuleBinding(
+    const TurboModuleProviderFunctionType &moduleProvider)
+    : moduleProvider_(moduleProvider) {}
+
+void TurboModuleBinding::install(
+    jsi::Runtime &runtime,
+    std::shared_ptr<TurboModuleBinding> binding) {
+  runtime.global().setProperty(
+      runtime,
+      "__turboModuleProxy",
+      jsi::Function::createFromHostFunction(
+          runtime,
+          jsi::PropNameID::forAscii(runtime, "__turboModuleProxy"),
+          1,
```

Comment:
what's this 1? (add a comment)

Reply:
This file was from facebook (with a patch from us) but it's gone now anyway.